### DISCUSSION TITLE
After initial sync files copied back.

### DISCUSSION THREAD

**Discussion by sytone** [remotely-save/remotely-save#144] [General]
I have noticed that after the initial sync a lot of files are uploaded again. I am assuming that the local modification time is now more recent than remote as it was saved locally. So the file has not changed but the time stamps have?  I have not had a chance to walk the code yet to confirm. 

Running webdav as the backend this time but also saw this with OneDrive. Running on Windows, Android and iOS

Anyone else seeing this?

**Comment by fyears** [comment]
the new version should fix many bugs like this
